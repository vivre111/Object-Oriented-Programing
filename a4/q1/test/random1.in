a b + 3 * 4 5 4 / - + done
print 
set a 3
print
