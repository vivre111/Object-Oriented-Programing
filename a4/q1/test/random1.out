(((a + b) * 3) + (4 - (5 / 4)))
(((a + b) * 3) + (4 - (5 / 4)))
(((3 + b) * 3) + (4 - (5 / 4)))
